core/views.py

```diff
@@ -144,3 +148,82 @@ def download_file(request, model, id, field):
     if len(file) is not None:
         response['Content-Length'] = len(file)
     return response
+
+
+def get_filter_fields(model):
+    """
+    Return a list of all filterable fields of Model.
+
+    -Takes: Model type
+    -Returns: A list of fields as string (excluding ID and file uploading)
+    """
+    model_fields = model._meta.get_fields()
+    fields = []
+    for field in model_fields:
+        res = str(field).split('.')
+        if res[1] == model.__name__ and not isinstance(field, (FileField, AutoField)):
```

Comment:
We probably DO want to serialize the file fields in some manner (though this can be a future PR).  Specifically, we want to at least mention that the file exists so we could use the download endpoint to fetch it (and, ultimately, also have a way to redirect to get it directly from S3 when appropriate).  For PUT/POST operations, we want to expose the ability to upload new files, too.